controllers:
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 100
  legged_controller:
    type: legged/LeggedController
  legged_cheater_controller:
    type: legged/LeggedCheaterController
